segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkManagerConfig.java

```diff
@@ -16,13 +16,13 @@
 import lombok.NonNull;
 
 /**
- * Configuration for {@link ChunkManager}.
+ * Configuration for {@link ChunkedSegmentStorage}.
  */
 @AllArgsConstructor
 @Builder(toBuilder = true)
 public class ChunkManagerConfig {
     /**
-     * Default configuration for {@link ChunkManager}.
+     * Default configuration for {@link ChunkedSegmentStorage}.
      */
     public static final ChunkManagerConfig DEFAULT_CONFIG = ChunkManagerConfig.builder()
```

Comment:
Don't we need to rename this to `ChunkedSegmentStorageConfig`?

Reply:
Sorry missed that.
Fixed.
https://github.com/pravega/pravega/pull/4686/commits/eda93411b4324c1a76ab2a1629b6a93ac947c9f5